csharp/ql/src/experimental/CWE-918/RequestForgery.ql

```diff
@@ -15,5 +15,5 @@
 
 from RequestForgeryConfiguration c, DataFlow::PathNode source, DataFlow::PathNode sink
 where c.hasFlowPath(source, sink)
-select sink.getNode(), source, sink, "$@ flows to here and is used in a server side web request.",
-  source.getNode(), "User-provided value"
+select sink.getNode(), source, sink, "The URL of this request depends on a $@.", source.getNode(),
```

Comment:
Looks legit?

Reply:
Yes, that's an FP from the QL-for-QL check.  
But that because this query uses a DataFlow configuration, but the configuration also includes most of the taint-tracking steps, so the configuration is effectively a TaintTracking query.

Reply:
Ah, ok.